lib/locreator.js

```diff
@@ -75,6 +75,12 @@ Locreator.prototype.addStarMethods = function locreator(locatorId, _locator, par
   locatorObject[locatorId + 'OptionValue'] = function (optionValue) {
     return drivex.selectByOptionValue(locator, optionValue, parentWebElement);
   };
+  locatorObject[locatorId + 'ValidateText'] = function (ValidateText) {
```

Comment:
can you change the name of this argument to `comparisonText` or something else that succinctly describes its purpose?